addons/gui/stringtable.xml

```diff
@@ -102,6 +102,7 @@
             <Spanish>CABEZA - SIN LINEA</Spanish>
             <Japanese>頭部 - ライン無し</Japanese>
             <Italian>TESTA - SENZA LINEA</Italian>
+            <German>KOPF – KEINE LINIE</German>
```

Comment:
Is this related to IV-Lines? If so, a better translation would be "KOPF - KEIN ZUGANG", which would also apply to each following instance.